cmd/fts/cmd/staging.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"fmt"
+	"sync"
+)
+
+type StagingStatus struct {
+	cachedBytes     int64
+	maxStagingBytes int64
+	minDealBytes    int64
+	m               sync.Mutex
+}
+
+func (sc *StagingStatus) Ready(bytes int64) (bool, error) {
```

Comment:
same as previous commit, still just tracks available resources on the remote staging

Reply:
So just want to understand this... you're keeping a local representation of some known server side limit for the amount of data that can be staged at once? Idea is to just never get to the point where the server side is borked because it's out of disk space?

Reply:
And I guess we're assuming that `fts` has exclusive access to the powergate instance

Reply:
first one is correct. second one is 1/2 correct. i was actually wondering if we should make it native in the admin api to set staging limits on ffs instances. there are some edge cases that make this hard (add by cid is one jsign already brought up), but it still seems better than doing it client side. 

long run, the default mode for this would be to do it over buckets. at which point, you can actually let the user set the limit to whatever they want. so didn't get to bogged down here for now.

Reply:
i think it _would_ be very nice to test this out with some users on a single POW where we give them their own token though... something to talk about

Reply:
Yea, long term doing any type of limiting server side is better, however it is done. From the client, it would just be a simple error you might get back from the call to `Stage`.